Workiva/w-mobile-kit
# MOBILE-1730: Populate Wdesk Users Info in Autocomplete

## Description

Need the auto complete component to search specifically for Wdesk Users
## What Was Changed

Changes were made in mobile kit to accommodate cases or customization options not thought of. The delegate structuring was changed a little bit so that the datasource protocol is what populates the auto complete table, the delegate is what accepts the selection of auto complete cells. A bug fix around the keyboard height was resolved as well when shortcuts or custom keyboards are enabled.
## Testing
- Ensure all unit tests pass
- Ensure code coverage is still acceptable with new changes
- Ensure the flow of creating the auto complete view in the example code is easy to understand
- Ensure the view is always on top of the keyboard properly regardless of having shortcuts or predictions enabled

---

Please Review: @Workiva/mobile
